pulpcore/pulpcore/app/models/repository.py

```diff
@@ -239,6 +239,8 @@ class RepositoryVersion(Model):
             1 + the most recent version.
         created (models.DateTimeField): When the version was created.
         action  (models.TextField): The action that produced the version.
+        complete (models.BooleanField): False indicates that the Repository Version is still being
```

Comment:
I think this description needs to state that the field indicates whether or not the Repository Version is user visible. Then you can say what the False means